backend/mora/mapping.py

```diff
@@ -34,6 +34,7 @@
 LOCATION = 'location'
 ERROR = 'error'
 USER_SETTINGS = 'user_settings'
+INTEGRATIONDATA = 'integrationdata'
```

Comment:
Igen, et `_` ville være bedre.

Reply:
In this case, I agree — the other variables are in English :)